rpc/contract_test.go

```diff
@@ -131,27 +171,17 @@ func TestClassHashAt(t *testing.T) {
 	}[testEnv]
 
 	for _, test := range testSet {
-
+		require := require.New(t)
 		spy := NewSpy(testConfig.provider.c)
 		testConfig.provider.c = spy
 		classhash, err := testConfig.provider.ClassHashAt(context.Background(), WithBlockTag("latest"), test.ContractHash)
-		if err != nil {
-			t.Fatal(err)
-		}
+		require.NoError(err)
+		require.NotEmpty(classhash, "should return a class")
+		require.Equal(test.ExpectedClassHash, classhash)
+
 		diff, err := spy.Compare(classhash, false)
-		if err != nil {
-			t.Fatal("expecting to match", err)
-		}
-		if diff != "FullMatch" {
-			if _, err := spy.Compare(classhash, true); err != nil {
-				log.Fatal(err)
-			}
-			t.Fatal("structure expecting to be FullMatch, instead", diff)
-		}
-		if classhash == nil {
-			t.Fatalf("should return a class, instead %v", classhash)
-		}
-		require.Equal(t, test.ExpectedClassHash, classhash)
+		require.NoError(err, "expecting to match")
+		require.Equal(diff, "FullMatch", "structure expecting to be FullMatch")
```

Comment:
What is this doing? Does it test anything that require.Equal(test.ExpectedClassHash, classhash) doesn't?

Reply:
I have the same thought. Removed in the new commit